packages/client/src/pages/LeaderBoard/leaderBoard.module.css

```
@@ -0,0 +1,48 @@
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.title {
+  color: var(--color-accent-secondary);
+  margin-top: 80px;
+}
+
+.line {
+  width: 728px;
+  height: 3px;
+  border-radius: 28px;
+  background: rgb(204, 204, 204);
```

```suggestion
  background: var(--color-side);
```
Еще для темной темы нужно добавить непрозрачность 40%. Пример в форуме можешь глянуть. Там такая запись `:global(.app[data-theme="dark"]) { ... }`